lib/src/extensions/application.dart

```diff
@@ -21,3 +22,11 @@ extension ApplicationExtensions on PartialApplication {
         },
       );
 }
+
+extension ApplicationExtensions on Application {
+  /// The URL of this application's icon image.
+  Uri? iconUrl({CdnFormat? format, int? size}) => icon?.get(format: format, size: size);
+
+  /// The URL of this application's cover image.
+  Uri? coverUrl({CdnFormat? format, int? size}) => coverImage?.get(format: format, size: size);
+}
```

Comment:
What do these methods improve over simply calling `application.coverImage?.get()`? I don't think `application.coverUrl()` is really worth having the whole extra extension.

Same for all the other URL methods.